app/src/org/storymaker/app/HomeActivity.java

```diff
@@ -87,6 +87,15 @@
     public void onCreate(Bundle savedInstanceState) {
     	super.onCreate(savedInstanceState);
 
+        // set title bar as a reminder if test server is specified
```

Comment:
yeah, we should probably get the title and dynamically stuff BETA in there... maybe even at the start so it shows up if there are long titles on small screens.  

"BETA: StoryMaker 2"